src/albums/albums.controller.ts

```diff
@@ -46,7 +39,9 @@ export class AlbumController {
     image: Express.Multer.File,
     @Body() createAlbumDto: CreateAlbumDto,
   ) {
-    return await this.albumService.create(createAlbumDto, image);
+    image.originalname = getFileName(image)
+    const url = await this.albumService.upload(image);
```

Comment:
no , Transfer the file directly to the controller , and write all logic to service